inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -15,55 +15,58 @@
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertInterpolate1ToInterpolate4, "ConvertInterpolate1ToInterpolate4", 0);
 
 ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolate4() {
-    auto interpolate1 = ngraph::pattern::wrap_type<ngraph::opset1::Interpolate>({pattern::any_input(pattern::has_static_rank()), pattern::any_input()});
+    auto interpolate1 = ngraph::pattern::wrap_type<ngraph::opset1::Interpolate>({ngraph::pattern::any_input(), ngraph::pattern::any_input()});
```

Comment:
nit: I didn't understand why you added the `ngraph::` namespace. It worked before but `ngraph::` prefix makes this line longer.

Reply:
Fixed.